### DISCUSSION TITLE
Is it possible to train each model of Auto3dSeg on a different machine separately?

### DISCUSSION THREAD

**Discussion by thangngoc89** [Project-MONAI/MONAI#6988] [General]
I have several GPUs attached to different machines. These GPUs could not be put together in a single machine.
Since Auto3dSeg trains several models, can I move some trainings to a different machines?

**Comment by KumoLiu** [comment]
Hi @thangngoc89, you can set `num_nodes` in `set_device_info` to run multi-node training.
https://github.com/Project-MONAI/MONAI/blob/73a760163cc61e84de87f21b31434dac65000d9a/monai/apps/auto3dseg/auto_runner.py#L491

Thanks!
